tests/test_isrTaskLSST.py

```diff
@@ -476,6 +476,79 @@ def test_isrFlat(self):
         ratio = result2.exposure.image.array / result.exposure.image.array
         self.assertFloatsAlmostEqual(ratio, flat_nodefects.image.array, atol=1e-4)
 
+    def test_isrBrighterFatter(self):
+        """Test processing of a flat frame."""
+        mock_config = self.get_mock_config_no_signal()
+        mock_config.doAddDark = True
+        mock_config.doAddFlat = True
+        # The doAddSky option adds the equivalent of flat-field flux.
+        mock_config.doAddSky = True
```

Comment:
In this test we aren't adding a source; is that intentional?

Reply:
I found that the BF correction (even in it's best state) isn't correct perfect enough to pass the test cases in the skyImage test without massively opening up the assert validity range to the point that they really aren't testing anything. So I formed this new test (which is also very fast to run < 10s), which sees if the BF correction can reconstruct poisson variations in a flat image.

Reply:
I haven't run the test myself, but is it passing whether or not we apply the BF correction?  If so, then I fear it's too trivial.  It may be worth (a) adding the source; (b) doing the comparison below away from the source; (c) at least doing a simple specific test of the size of the source before or after BF correction and confirming that it is closer to the correct/true width after correction.

Reply:
The test failed without the BF correction turned on. But the test you describe seems more stable. I will move it towards that.

Reply:
Updated the test.